### PR TITLE
Pazuz-ready changes

### PR DESCRIPTION
**Pull request by chaotixkilla**
Absorbs the following:
https://github.com/lamassu/lamassu-server/pull/850
https://github.com/lamassu/lamassu-server/pull/849
https://github.com/lamassu/lamassu-server/pull/848
https://github.com/lamassu/lamassu-server/pull/847
https://github.com/lamassu/lamassu-server/pull/846

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Get the project running on a droplet (stage 1)](https://trello.com/c/4Jvjm70n/957-get-the-project-running-on-a-droplet-stage-1)

File: lib/db.js [hunk 1]
```diff
@@ -36,7 +37,10 @@ const _task = (obj, opts, cb) => {
   })
 }
 
-const getSchema = () => 'public'
+const getSchema = () => {
+  const store = asyncLocalStorage.getStore() ?? defaultStore()
```
Comment (RafaelTaranto) [inline lib/db.js:42]: I don't think we should go to defaultStore here. If the storage isn't set at this point i think the query should fail fast on lamassu so we can spot it before we get silent issues in pazuz

File: lib/new-admin/middlewares/session.js [hunk 2]
```diff
@@ -19,12 +20,16 @@ const getSecret = () => {
 
 const hostname = options.hostname
 
+const lamaDb = {
+  query: (query, values, qrm) => db.query(query, values, qrm, false)
+}
```
Comment (RafaelTaranto) [inline lib/new-admin/middlewares/session.js]: What's this doing?

File: new-lamassu-admin/src/pages/Authentication/Input2FAState.js [hunk 3]
```diff
@@ -56,7 +58,21 @@ const Input2FAState = ({ state, dispatch }) => {
...
+          context: {
+            headers: {
+              'Pazuz-Operator-Identifier': base64.encode(state.clientField)
+            }
+          }
+        }
+        return getUserData(
+          process.env.REACT_APP_BUILD_TARGET === 'LAMASSU'
+            ? R.omit(['context'], options)
+            : options
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Authentication/Input2FAState.js]: Is there harm sending in the context?

File: new-lamassu-admin/src/pages/UserManagement/modals/CreateUserModal.js [hunk 4]
```diff
@@ -74,7 +75,11 @@ const CreateUserModal = ({ state, dispatch }) => {
 
   const [createUser, { error }] = useMutation(CREATE_USER, {
     onCompleted: ({ createRegisterToken: token }) => {
-      setCreateUserURL(urlResolver(`/register?t=${token.token}`))
+      const queryParams =
+        process.env.REACT_APP_BUILD_TARGET === 'LAMASSU'
+          ? `t=${token.token}`
+          : `t=${token.token}&id=${base64.encode(usernameField)}`
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/UserManagement/modals/CreateUserModal.js:82]: What does the ID does on the server side for this one? Might be a good idea to leave a comment on this

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: lib/new-admin/middlewares/session.js (hunk 2 above)
Comment (chaotixkilla) [reply lib/new-admin/middlewares/session.js]: I believe it's older code from this PR that I haven't reverted

**Pull request closed without merging**